src/test/kotlin/com/rooplor/classcraftbackend/controllers/AuthControllerTest.kt

```diff
@@ -1,45 +1,60 @@
 package com.rooplor.classcraftbackend.controllers
 
+import com.rooplor.classcraftbackend.configs.TestConfig
+import com.rooplor.classcraftbackend.dtos.Token
 import com.rooplor.classcraftbackend.services.AuthService
+import com.rooplor.classcraftbackend.services.cookie.CookieService
 import org.junit.jupiter.api.Test
-import org.mockito.Mockito.any
 import org.mockito.Mockito.`when`
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
 import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.context.annotation.Import
 import org.springframework.http.MediaType
 import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post
 
 @WebMvcTest(AuthController::class)
+@Import(TestConfig::class)
 class AuthControllerTest {
     @Autowired
     private lateinit var mockMvc: MockMvc
 
     @MockBean
     private lateinit var authService: AuthService
 
+    @MockBean
+    private lateinit var cookieService: CookieService
+
     @Test
     fun `test validateToken valid`() {
         val idToken = "validToken"
-        `when`(authService.validateIdToken(idToken)).thenReturn(any())
+        val accessToken = "accessToken"
+        val refreshToken = "refreshToken"
+        val tokenResponse = Token(accessToken, refreshToken)
+
+        `when`(authService.login(idToken)).thenReturn(tokenResponse)
+        `when`(cookieService.createCookie("accessToken", accessToken, 36000)).thenReturn(null)
+        `when`(cookieService.createCookie("refreshToken", refreshToken, 604800)).thenReturn(null)
```

Comment:
Should we use format 10 * 60 * 60 * 100 ?

Reply:
done, It should be 10 * 60 * 60 because cookie age unit is sec.